Capstone/Profile/AccountViewController.m

```diff
@@ -32,6 +32,25 @@ - (void)viewDidLoad {
     self.tableView.dataSource = self;
     [self setUser];
     self->currentUser = [PFUser currentUser];
+    if (self->currentUser == nil) {
+        [self setUser];
+            dispatch_async(dispatch_get_main_queue(), ^{
+                [FBSDKProfile loadCurrentProfileWithCompletion:^(FBSDKProfile * _Nullable profile, NSError * _Nullable error) {
+                    if(profile) {
+                        NSString *lastnameWithSpace = [@" " stringByAppendingString:profile.lastName];
+                        NSString *fullName = [profile.firstName stringByAppendingString:lastnameWithSpace];
+                        self.profileName.text = fullName;
+                        NSURL *url = [profile imageURLForPictureMode:FBSDKProfilePictureModeSquare size:CGSizeMake(0, 0)];
+                        UIImage *image = [UIImage imageWithData:[NSData dataWithContentsOfURL:url]];
+                        self.profileImage.image = image;
+                    }
+                }];
+            });
+        UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Create an account!" message:@"Please create an account to have access to matching, news, and likes." preferredStyle:(UIAlertControllerStyleAlert)];
```

Comment:
Let's create string constants for these titles! For the sake of readability and reducing how long statements are
```suggestion
NSString *facebookLoginAlertTitle = @"Create an account!";
NSString *facebookLoginAlertMessage = @"Please create an account to have access to matching, news, and likes.";
 UIAlertController *alert = [UIAlertController alertControllerWithTitle:facebookLoginAlertTitle message:facebookLoginAlertMessage preferredStyle:(UIAlertControllerStyleAlert)];

Reply:
resolved